microservices/user/src/services/user_services.py

```diff
@@ -66,6 +74,65 @@ def create_client_service(client_data):
             delete_auth_user_service(auth_user["id"])
         raise
 
+def imports_users_service(user_id, data):
+
+    # Obtener idCompany del cliente en sesión
+    user_repository = UserRepository()
+    user_repository.set_serializer(serializer_user_class)
+    user = user_repository.get_by_field("auth_id", user_id)
+
+    
+    for row in data:
+        if data.line_num == 1:
+            continue
+
+        #Crear Auth de los usuarios
+        auth_data = {"email": row[2], "role": "user", "password": row[3]}
+        url = get_request_url("auth")
+        try:
+            auth_user = send_request("POST", url, auth_data)["data"]
+        except Exception as e:
+            logger.error(ExceptionsMessages.RESOURCE_EXISTS.value + ": " + row[2])
+            raise CustomException(ExceptionsMessages.RESOURCE_EXISTS.value + " : " + row[2])
+        
+        user_data = {
+            "name": row[0],
+            "phone": row[1],
+            "company_id": user["company_id"],
+            "auth_id": auth_user["id"],
+            "importance": int(row[4]),
+            "dni": row[3],
+            "channel": UserChannel.EMAIL.value
+        }
+        user = create_user_client_service(user_data)
+
+        products = get_products_service(user["company_id"])
+
+        for item in products:
+            data_product_user = {
+                "id_user": user["id"],
+                "product_id": item["id"]
+            }
+            data_new_product_user = ProductUserCreateSerializer().load(data_product_user)
+            
+            product_user_repository = ProductUserRepository()
```

Comment:
Saca esta instancia del ciclo for. No es necesario que crees una nueva conexión por cada usuario que vas a agregar